api/envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -68,90 +74,70 @@ message Policy {
   repeated Principal principals = 2 [(validate.rules).repeated .min_items = 1];
```

Comment:
It seems that there is space between "repeated" and ".min_items"? Should it be removed?

Reply:
The clang-format rules doesn't distinguish between the option name and the `repeated` decorator for a field. Unfortunately this has to remain but it's fortunately compatible with protoc